controllers/nodefeaturediscovery_status.go

```diff
@@ -30,8 +30,9 @@ import (
 )
 
 const (
-	nfdWorkerApp string = "nfd-worker"
-	nfdMasterApp string = "nfd-master"
+	nfdWorkerApp   string = "nfd-worker"
+	nfdMasterApp   string = "nfd-master"
+	nfdTopologyApp string = "nfd-topology-updater"
```

Comment:
nit: just for consistency 😄 

```suggestion
	nfdTopologyUpdaterApp string = "nfd-topology-updater"
```

Reply:
I love how a small change like this breaks the entire ci